src/main/java/dinkplugin/notifiers/data/PkNotificationData.java

```diff
@@ -0,0 +1,47 @@
+package dinkplugin.notifiers.data;
+
+import dinkplugin.message.Field;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.api.kit.KitType;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class PkNotificationData extends NotificationData {
+
+    String name;
+    int combatLevel;
+    Map<KitType, SerializedItemStack> equipment;
```

Comment:
Should these be named differently to make it clearer that they refer to the victim's info?

Reply:
Yeah, `victimEquipment` seems reasonable. Remember to update the readme at the same time